laws/src/main/scala/org/http4s/laws/discipline/arbitrary.scala

```diff
@@ -203,13 +203,14 @@ private[discipline] trait ArbitraryInstances { this: ArbitraryInstancesBinCompat
       } yield Query(vs: _*)
     }
 
-  implicit val http4sTestingArbitraryForHttpVersion: Arbitrary[HttpVersion] =
-    Arbitrary {
-      for {
-        major <- choose(0, 9)
-        minor <- choose(0, 9)
-      } yield HttpVersion.fromVersion(major, minor).yolo
-    }
+  implicit val http4sTestingArbitraryForHttpVersion: Arbitrary[HttpVersion] = {
+    val genSpecified = Gen.oneOf(HttpVersion.specified)
```

Comment:
Focus on creating real ones.  Most of the 100 will never be seen in the real world.